owncloudDomain/src/main/java/com/owncloud/android/domain/authentication/usecases/LoginOAuthAsyncUseCase.kt

```diff
@@ -29,7 +29,6 @@ class LoginOAuthAsyncUseCase(
 
     override fun run(params: Params): String {
         require(params.serverInfo != null) { "Invalid server info" }
-        require(params.username.isNotEmpty()) { "Invalid username" }
```

Comment:
Just curiosity, why username is not required?

Reply:
Because of the line of your previous comment 😄 

`tokenResponse.additionalParameters[AccountUtils.Constants.KEY_USER_ID]?.let { it } ?: "",`

Reply:
Yes, i mean why is not required now and it was required previously

Reply:
Because we always had a username before... it's something that appeared with oidc servers